backend/src/src-projects/src/main/java/com/siemens/sw360/projects/ProjectHandler.java

```diff
@@ -53,7 +53,9 @@
 
     @Override
     public List<Project> refineSearch(String text, Map<String, Set<String>> subQueryRestrictions,User user) throws TException {
-        return searchHandler.search(text, subQueryRestrictions,user);
+        List<Project> foundProjects = searchHandler.search(text, subQueryRestrictions,user);
+        foundProjects.stream().forEach(p -> p.releaseIds = p.isSetReleaseIdToUsage()? p.releaseIdToUsage.keySet() : Collections.EMPTY_SET);
```

Comment:
for readability one should not use the ternary operator here and/or format this better